doc/holochain_101/src/distributed_hash_table.md

```diff
@@ -1 +1,33 @@
 # Distributed Hash Table
+
+## Local Hash Table
+
+### implementation details
+
+First, read about [state actors](/state/actors.html).
+
+The 1:1 API implementation between actors and their inner table is achieved by
+internally blocking on an `ask` from riker patterns.
+
+https://github.com/riker-rs/riker-patterns
+
+The actor ref methods implementing `HashTable` sends messages to itself.
+
+Calling `table_actor_ref.commit(entry)` looks like this:
+
+0. the actor ref constructs a `HashTableProtocol::Commit` message including the entry
+0. the actor ref calls its own `ask` method, which builds a future using riker's `ask`
+0. the actor ref blocks on its internal future
+0. the referenced actor receives the `Commit` message and matches/destructures this into the entry
+0. the entry is passed to the `commit()` method of the inner table
```

Comment:
I am a bit concerned of all those `commit()`s here which actually should be `put()`s.
I've created ticket #274 as a follow up task to make these function names match the mental modal we have...